issues.md

```diff
@@ -0,0 +1,36 @@
+Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION].
```

Comment:
Please add some functionality to check on an update for this file when a commit happens to CCExtractor. We already get a notification of new commits, so it should be doable to check if this file was modified. I wouldn't want to have to update this file each time.